### ISSUE TITLE
Links to reviewer reports in published papers

### ISSUE THREAD

**Issue by pneuvial** [computorg/computo-quarto-extension#16]
The template currently assumes that

- there are exactly two referees
- their reports correspond to issues 1 and 2 of the manuscript repository

cf https://github.com/computorg/computo-quarto-extension/blob/main/_extensions/computo/partials/html/title-metadata.html#L89-L93

Both are false in the case of https://computo.sfds.asso.fr/published-202306-sanou-multiscale_glasso/.
Therefore, a link to the third review is missing, and the links to the other two reviews are broken.

I guess the way to fix this is to add more info in the header of the notebook, e.g.

```
published: 
    reviewer-1: 3 # id of issue corresponding to the report of reviewer 1 in the manuscript repo
    reviewer-2: 4
    reviewer-3: 5
```
and make use of it in the metadata layout, with the default corresponding 

```
published: 
    reviewer-1: 1
    reviewer-2: 2
```

Can someone handle this?

**Comment by pneuvial** [comment]
Fixed by #17

**Comment by pneuvial** [comment]
Note that the badge "reviews|reports" is a bit awkward, but at least the link leads to the reports, cf https://computo.sfds.asso.fr/published-202306-sanou-multiscale_glasso/
